server/ec2spotmanager/views.py

```diff
@@ -407,14 +408,14 @@ def createConfig(request):
 
 
 @deny_restricted_users
-def editConfig(request, configid):
-    config = get_object_or_404(PoolConfiguration, pk=configid)
+def editEc2Config(request, configid):
```

Comment:
This URL and handler should be generic, but use the type of the configuration object to decide which configuration class to use.